report_generator/conference.py

```diff
@@ -0,0 +1,40 @@
+class Conference:
+    def __init__(self, year, report_yaml, talks_df, p_talks_df):
```

Comment:
What is the different between `talk_df` and `p_talk_df`?

Reply:
talk_df: accepted talk information in dataframe
p_talk_df: proposed talk  information in dataframe

Reply:
I'd suggest using clear variable names like `accepted_talk_df` and `proposed_talk_df`. Or we should at this have docstring